packages/core/interfaces/src/compilerSupport/aquaTypeDefinitions.ts

```diff
@@ -233,12 +229,12 @@ export interface ServiceDef {
     /**
      * Default service id. If the service has no default id the value should be undefined
      */
-    defaultServiceId?: string;
+    defaultServiceId?: {};
```

Comment:
it's not recommended to ever use `{}` type
also it's weird that defaultServiceId is `{}` and not string

Reply:
Actually there is a reason for that. In air definition we get an object with this definition
```
interface DefaultServiceId {
    s_Some__f_value?: string
}
```

Reply:
But for simplicity we can ignore this detail and allow user to work with this property as a plain string.
I revert type to "string"